appGeo/src/app/chat/chat.component.spec.ts

```diff
@@ -1,24 +1,24 @@
-/* import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
I'm very confused by what's going on in a lot of these where they're deleting the entire file and readding them. It's hard to write the review because I can't tell what code is being changed and what's not.

(This is marked as request changes just because I would like to know what's going on before it's approved.)

Reply:
yeah that's just a ng generated file, in the other branch with the cleaning up of files I deleted this because all our testing should be done in the `tests` folder anyways

Reply:
but it's marked as being added again down before?

Reply:
I think that happened when I merged with main. I did the deletion of that file in a different branch so maybe when I switched branches it wanted the file to be in this branch and readded it? Not sure why else it would do that